components/Extras/Extra-Components/Navigation.css

```diff
@@ -0,0 +1,74 @@
+.navigation li a{
+    text-decoration: none;
+    color: white;
+}
+
+.navigation li {
+    display: inline-block;
+    margin: 0.5rem 1rem;
+}
+.navigation ul {
+    background-color: rgb(77, 76, 76);
+}
+
+.navigation li a:hover {
+    color: rgb(167, 214, 255);
+    text-decoration: underline;
+}
+
+.navigation-vertical li a{
```

Comment:
Avoid grabbing elements by their name. Use class in place of it.

Reply:
OKAY WILL DO SO

Reply:
Changes are done